src/Verify.Terminal/Commands/ReviewCommand.cs

```diff
@@ -81,41 +86,38 @@ public sealed class Settings : CommandSettings
     }
 
     private static SnapshotAction ShowPrompt()
+    {
```

Comment:
I would also want this functionality to be a fallback method instead of replacing what we currently have. We can check if the input has been redirected, and fallback to this new functionality.

Reply:
@patriksvensson Sure, this is definitely possible to do - technically speaking that is -, but are you sure this is a good idea? 🤔  There are many consoles that redirect the output, including the terminal in Visual Studio Code... So a user could easily have different UX using this tool **on the same machine** depending on the terminal they happen to be using each time... Seems a little odd if you ask me.

Also, using `ReadLine` for prompting for an answer is the approach we see on popular command-line tools like Git, Homebrew, npm, AWS Cli, Azure Cli, and many others... Can you think of any modern/popular command-line tools that use `ReadKey`? I suspect there aren't many because of this exact limitation on input redirection...

Reply:
- Yes, "Insta" which I based this tool on, for one.

I don't want to have to press enter for each file when I review them, so if stdin isn't redirected (the usual case when running an interactive app), I want to grab input without having to press enter if possible.